app/views/admin/service_orders/edit.html.slim

```diff
@@ -9,6 +9,8 @@
     = f.input :her_story
     = f.input :our_relationship
     = f.input :your_name
+    = f.input :status, priority: [:pending], collection: [:pending, :approve, :reject]
```

Comment:
Should get this data from Model instead. So in the future, if you need to change status item, just go to model and edit once.